Documentation/ceph-upgrade.md

```diff
@@ -140,156 +149,13 @@ POD_NAME=$(kubectl -n $ROOK_NAMESPACE get pod -o custom-columns=name:.metadata.n
 kubectl -n $ROOK_NAMESPACE get pod ${POD_NAME} -o jsonpath='{.spec.containers[0].image}'
 ```
 
-### All Pods Status and Version
```

Comment:
What if we change the `1.0` instances to `1.1` instead of deleting all this. Most (but certainly not all) of the steps here will be helpful to have for the next doc update, even if just as a basic skeleton.

Reply:
Yeah I was debating with myself about this as well. My thought was to make it as clear as possible that upgrades to master are not supported and when it is closer to the 1.1 release we would essentially add this content back. If we change it now to 1.1, it will be a while before it's applicable, but I see that it wouldn't hurt as long as we're clear about not supporting master upgrades.